tests/components/sonos/test_sensor.py

```diff
@@ -123,3 +123,14 @@ async def test_device_payload_without_battery_and_ignored_keys(
     await hass.async_block_till_done()
 
     assert ignored_payload not in caplog.text
+
+
+async def test_audio_input_sensor(hass, config_entry, config, soco):
+    """Test sonos device with battery state."""
+    await setup_platform(hass, config_entry, config)
+
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
```

Comment:
`async_get_registry` is legacy. Use `async_get` instead. And we try to avoid accessing `hass.helpers`.

Reply:
Addressed as part of #61198.